src/components/Auth/utils.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react"
+import { Auth } from "aws-amplify"
+
+import { User } from "./SignIn"
+
+export async function signUp(userInput: User) {
+  const { username, password } = userInput
+  try {
+    const { user } = await Auth.signUp({
+      username,
+      password,
+      attributes: {},
+    })
+    return <p> {user} </p>
+  } catch (error) {
+    ;<p> Error: {error} </p>
```

Comment:
As mentioned in the overview for this PR, we need to create a Message/Alert component that relays feedback (i.e Success ||  Error) to the user